### PR TITLE
secrets: add Azure KeyVault Example

### PR DESCRIPTION
**Pull request by vsaroopchand**
Example test for #1315 ; adds to #76

### CODE REVIEW

File: secrets/azurekeyvault/example_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	)
+
+	// Now we can use keeper to encrypt or decrypt.
+	ctx := context.Background()
+	plaintext := []byte("Hello, Secrets!")
+	ciphertext, err := keeper.Encrypt(ctx, plaintext)
+	if err != nil {
+		log.Fatal(err)
+	}
+	decrypted, err := keeper.Decrypt(ctx, ciphertext)
```
Comment (shantuo) [inline secrets/azurekeyvault/example_test.go]: I think you need to assign the `err` to blank or handle it, either way is fine.

File: secrets/azurekeyvault/example_test.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	// - client: *keyvault.BaseClient instance, see https://godoc.org/github.com/Azure/azure-sdk-for-go/services/keyvault/v7.0/keyvault#BaseClient
+	// - keyVaultName: string representing the KeyVault name, see https://docs.microsoft.com/en-us/azure/key-vault/common-parameters-and-headers
+	// - keyName: string representing the keyName, see https://docs.microsoft.com/en-us/rest/api/keyvault/encrypt/encrypt#uri-parameters
+	// - keyVersion: string representing the keyVersion, see https://docs.microsoft.com/en-us/rest/api/keyvault/encrypt/encrypt#uri-parameters
+	// - opts: *KeeperOptions with the desired Algorithm to use for operations. See this link for more info: https://docs.microsoft.com/en-us/rest/api/keyvault/encrypt/encrypt#jsonwebkeyencryptionalgorithm
+	keeper := akv.NewKeeper(
+		client,
+		"replace with keyVaultName",
+		"replace with keyName",
+		"replace with keyVersion",
```
Comment (shantuo) [inline secrets/azurekeyvault/example_test.go]: For simplicity of the example, let's use an empty keyVersion here. You could add a comment like `// replace with keyVersion if you don't want to use the default one`.

File: secrets/azurekeyvault/example_test.go (hunk 1 above)
Comment (vsaroopchand) [reply secrets/azurekeyvault/example_test.go]: Ok

File: secrets/azurekeyvault/example_test.go (hunk 2 above)
Comment (vsaroopchand) [reply secrets/azurekeyvault/example_test.go]: OK

**Pull request merged**